src/app/zap-templates/zcl/data-model/chip/energy-evse-cluster.xml

```diff
@@ -0,0 +1,221 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="Energy Management"/>
+  <enum name="StateEnum" type="enum8" apiMaturity="provisional">
+    <cluster code="0x0099"/>
+    <item name="NotPluggedIn" value="0x00"/>
+    <item name="PluggedInNoDemand" value="0x01"/>
+    <item name="PluggedInDemand" value="0x02"/>
+    <item name="PluggedInCharging" value="0x03"/>
+    <item name="PluggedInDischarging" value="0x04"/>
+    <item name="SessionEnding" value="0x05"/>
+    <item name="Fault" value="0x06"/>
+  </enum>
+  <enum name="SupplyStateEnum" type="enum8" apiMaturity="provisional">
+    <cluster code="0x0099"/>
+    <item name="Disabled" value="0x00"/>
+    <item name="ChargingEnabled" value="0x01"/>
+    <item name="DischargingEnabled" value="0x02"/>
+    <item name="DisabledError" value="0x03"/>
+    <item name="DisabledDiagnostics" value="0x04"/>
+  </enum>
+  <enum name="FaultStateEnum" type="enum8" apiMaturity="provisional">
+    <cluster code="0x0099"/>
+    <item name="NoError" value="0x00"/>
+    <item name="MeterFailure" value="0x01"/>
+    <item name="OverVoltage" value="0x02"/>
+    <item name="UnderVoltage" value="0x03"/>
+    <item name="OverCurrent" value="0x04"/>
+    <item name="ContactWetFailure" value="0x05"/>
+    <item name="ContactDryFailure" value="0x06"/>
+    <item name="GroundFault" value="0x07"/>
+    <item name="PowerLoss" value="0x08"/>
+    <item name="PowerQuality" value="0x09"/>
+    <item name="PilotShortCircuit" value="0x0A"/>
+    <item name="EmergencyStop" value="0x0B"/>
+    <item name="EVDisconnected" value="0x0C"/>
+    <item name="WrongPowerSupply" value="0x0D"/>
+    <item name="LiveNeutralSwap" value="0x0E"/>
+    <item name="OverTemperature" value="0x0F"/>
+    <item name="Other" value="0xFF"/>
+  </enum>
+  <enum name="EnergyTransferStoppedReasonEnum" type="enum8" apiMaturity="provisional">
+    <cluster code="0x0099"/>
+    <item name="EVStopped" value="0x00"/>
+    <item name="EVSEStopped" value="0x01"/>
+    <item name="Other" value="0x02"/>
```

Comment:
Should "other" be consistent here and be of the value 0xFF?

Reply:
Ironically I used to have it that way, and was asked to change it.

We may add more faults, but not sure we are likely to add more Reasons for the current to stop flowing.

https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7705/files/1ed466ee37e1284cbec563477d667233293216cd used to have 255